mycore-solr/src/main/java/org/mycore/solr/index/handlers/MCRSolrAbstractIndexHandler.java

```diff
@@ -40,7 +41,7 @@ public MCRSolrAbstractIndexHandler() {
 
     public MCRSolrAbstractIndexHandler(SolrClient solrClient) {
         this.solrClient = solrClient != null ? solrClient : MCRSolrClientFactory.getMainSolrClient();
-        this.commitWithin = -1;
+        this.commitWithin = MCRConfiguration2.getInt("MCR.Solr.commitWithIn").orElse(-1);
```

Comment:
Should rather use `.orElseThrow(() -> createConfigurationException("MCR.Solr.commitWithIn"))` instead of `.orElse(-1)`. We don't really want default values in code anymore.